main.js

```diff
@@ -0,0 +1,263 @@
+// create by d1y<chenhonzhou@gmail.com>
```

Comment:
1. *created
2. You will be attributed already by github to the people reading the code, the comment strikes me as slightly obnoxious and will likely scare potential contributors off